crates/c/src/lib.rs

```diff
@@ -35,6 +35,9 @@ pub struct Opts {
     /// Set component string encoding
     #[cfg_attr(feature = "clap", arg(long, default_value_t = StringEncoding::default()))]
     pub string_encoding: StringEncoding,
+    // Skip optional null pointer and boolean result argument signature flattening
+    #[cfg_attr(feature = "clap", arg(long, default_value_t = false))]
+    pub no_sig_flattening: bool,
```

Comment:
Would it be more precise and descriptive to name this flag as `no_option_ty_flattening`?